benchmark/linear_attention/plot_results.py

```
@@ -60,10 +63,13 @@ def get_backend_display_name(backend: str, cudnn_version: str = None) -> str:
     return base_name
 
 
-def generate_charts(df: pd.DataFrame, output_dir: Path, gpu_name: str = "", cudnn_version: str = None, variant: str = "gdn", batch_sizes: list = None) -> list:
+def generate_charts(
+    df: pd.DataFrame, output_dir: Path, gpu_name: str = "", cudnn_version: str = None, variant: str = "gdn", batch_sizes: list = None, x_axis: str = "seqlen"
```

Ruff (0.16.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)